src/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Snapshot test test App should render and match the Snapshot 1`] = `null`;
```

Comment:
- [ ] Kindly add more snapshot tests for other components(header, button, app) [Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md#:~:text=Use%20Jest%20snapshots%20to%20test%20the%20components.)